depdive/Cargo.toml

```diff
@@ -8,7 +8,6 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0.38" # error handling library
-tabled = "0.2.1"
```

Comment:
Whats the rationale for wanting to remove this?

Reply:
rationale is - There are too many fields for a pretty `tabled` output which is no better than just `json` output. In midpoint demo, I just chose a subset of the fields to that doesn't break in the monitor screen but that ain't a feature. I think we can make it a future work on how to present a higher-level abstraction of these data based on idea here. #103